src/app/drivingLicence/steps.js

```diff
@@ -2,14 +2,34 @@ const details = require("./controllers/details");
 const root = require("./controllers/root");
 const validate = require("./controllers/validate");
 const licenceIssuer = require("./controllers/licence-issuer");
+const checkYourDetails = require("./controllers/check-your-details");
 
 module.exports = {
   "/": {
     resetJourney: true,
     entryPoint: true,
     skip: true,
     controller: root,
-    next: "licence-issuer"
+    next: [
+      {
+        field: "isAuthSourceRoute",
+        value: true,
+        next: "/check-your-details"
+      },
+      "/licence-issuer"
+    ]
+  },
+  "/check-your-details": {
```

Comment:
Have we confirmed this cant be manually accessed when performing a non auth source route

Reply:
A general error screen is returned when carrying out a non-auth source journey and a manual attempt is made to reach _/check-your-details_.